frontend/src/components/healthEndpoint.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+import axios from 'axios';
+
+const HealthCheck = () => {
```

Comment:
Same situation here, Component name should match the filename. it makes files and components both easier to find, and keeps the codebase clean.

Reply:
Component now matches filename